Resizetizer.NT/ResizetizeSharedImages.cs

```diff
@@ -70,95 +66,50 @@ System.Threading.Tasks.Task DoExecute()
 
 			System.Threading.Tasks.Parallel.ForEach(images, img =>
 			{
-				if (img.IsAppIcon)
-				{
-					var appIconName = "appicon"; // Path.GetFileNameWithoutExtension(img.Filename);
-
-					// Generate the actual bitmap app icons themselves
-					var appIconDpis = DpiPath.GetAppIconDpis(PlatformType, appIconName);
-
-					Log.LogMessage(MessageImportance.Low, $"App Icon");
-
-					// Apple and Android have special additional files to generate for app icons
-					if (PlatformType == "android")
-					{
-						Log.LogMessage(MessageImportance.Low, $"Android Adaptive Icon Generator");
-
-						appIconName = appIconName.ToLowerInvariant();
-
-						var adaptiveIconGen = new AndroidAdaptiveIconGenerator(img, appIconName, IntermediateOutputPath, this);
-						var iconsGenerated = adaptiveIconGen.Generate();
-
-						foreach (var iconGenerated in iconsGenerated)
-							resizedImages.Add(iconGenerated);
-					}
-					else if (PlatformType == "ios")
-					{
-						Log.LogMessage(MessageImportance.Low, $"iOS Icon Assets Generator");
-					
-						var appleAssetGen = new AppleIconAssetsGenerator(img, appIconName, IntermediateOutputPath, appIconDpis, this);
-
-						var assetsGenerated = appleAssetGen.Generate();
-
-						foreach (var assetGenerated in assetsGenerated)
-							resizedImages.Add(assetGenerated);
-					}
-
-					Log.LogMessage(MessageImportance.Low, $"Generating App Icon Bitmaps for DPIs");
-
-					var appTool = new SkiaSharpAppIconTools(img, this);
-
-					Log.LogMessage(MessageImportance.Low, $"App Icon: Intermediate Path " + IntermediateOutputPath);
-
-					foreach (var dpi in appIconDpis)
-					{
-						Log.LogMessage(MessageImportance.Low, $"App Icon: " + dpi);
-
-						var destination = Resizer.GetFileDestination(img, dpi, IntermediateOutputPath)
-							.Replace("{name}", appIconName);
-						Log.LogMessage(MessageImportance.Low, $"App Icon Destination: " + destination);
-						appTool.Resize(dpi, Path.ChangeExtension(destination, ".png"));
-					}
-				}
-				else
+				try
```

Comment:
This try is needed to catch the exceptions in the parallel. They get swallowed and the build just fails.